### PR TITLE
[MOSIP-31977] From release to develop

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java [hunk 1]
```diff
@@ -56,6 +59,12 @@ public class RestApiClient {
 	/** The logger. */
 	private final Logger logger = RegProcessorLogger.getLogger(RestApiClient.class);
 
+	@Value("${registration.processor.httpclient.connections.max.per.host:20}")
+	private int maxConnectionPerRoute;
+
+	@Value("${registration.processor.httpclient.connections.max:100}")
+	private int totalMaxConnection;
+
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java]: Why do we need these variable here ? Seems like not used in the file.

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/logger/RegProcessorLogger.java [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+import ch.qos.logback.classic.LoggerContext;
+
+
+
+import java.util.stream.Collectors;
+import java.util.HashMap;
+import java.util.Map;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/logger/RegProcessorLogger.java]: These imports look like duplicates.

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/StageHealthCheckHandler.java [hunk 3]
```diff
@@ -351,23 +350,22 @@ public void senderHealthHandler(Promise<Status> promise, Vertx vertx, MosipEvent
...
+						promise.complete(Status.OK(result));
+					} else {
 						final JsonObject result = resultBuilder.create()
-								.add(HealthConstant.ERROR, e.getMessage()).build();
+								.add(HealthConstant.ERROR, healthCheckDto.getFailureReason()).build();
 						promise.complete(Status.KO(result));
 					}
+
+				} catch (Exception e) {
+					final JsonObject result = resultBuilder.create().add(HealthConstant.ERROR, e.getMessage()).build();
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/StageHealthCheckHandler.java:364]: Why do we have lot of these line break changes ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java (hunk 1 above)
Comment (sowmya695) [reply registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java]: Removed as it is not used in release branch also

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/StageHealthCheckHandler.java (hunk 3 above)
Comment (sowmya695) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/StageHealthCheckHandler.java:364]: In develop branch i have above line 2 lines,but in release its in one line so change is present

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/logger/RegProcessorLogger.java (hunk 2 above)
Comment (sowmya695) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/logger/RegProcessorLogger.java]: Removed and checked other classes also.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**